.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This shouldn't have been commited. Remove this from the git repo, and add .idea to your global .gitignore